File: crates/worldgen/src/throughput_resource_discovery_research_copilot.rs
//! Worldgen P05 AFA-worldgen-P05-F11 throughput research_copilot.
use super::resource_copilot_support::{self,ResourceCopilotRequest,ResourceCopilotReceipt};
pub const FEATURE_ID:&str="AFA-worldgen-P05-F11"; pub const CONTRACT_VERSION:&str="worldgen-throughput-resource-copilot/1.0";
pub fn worldgen_throughput_resource_discovery_research_copilot_manifest()->serde_json::Value{resource_copilot_support::manifest(FEATURE_ID,CONTRACT_VERSION,"ResourceCopilotRequest1@1","prospective high-throughput","A2")}
pub fn run_worldgen_throughput_resource_discovery_research_copilot(request:&ResourceCopilotRequest)->Result<ResourceCopilotReceipt,resource_copilot_support::ResourceCopilotError>{resource_copilot_support::run(request,FEATURE_ID,CONTRACT_VERSION,"prospective high-throughput",true,false)}
pub use resource_copilot_support::{ResourceCopilotError,ResourceCopilotReceipt as WorldgenthroughputResourceresearchcopilotReceipt,ResourceCopilotRequest as WorldgenthroughputResourceresearchcopilotRequest};

File: python/prism_sdk/throughput_resource_discovery_research_copilot.py
"""Worldgen P05 AFA-worldgen-P05-F11 product wrapper."""
from .worldgen_resource_copilot_support import ResourceCopilotRequest, ResourceCopilotReceipt, manifest as _manifest, run as _run
FEATURE_ID="AFA-worldgen-P05-F11"
CONTRACT_VERSION="worldgen-throughput-resource-copilot/1.0"
def worldgen_throughput_resource_discovery_research_copilot_manifest(): return _manifest(feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, input_schema="ResourceCopilotRequest1@1", scale="prospective high-throughput", autonomy_tier="A2")
def run_worldgen_throughput_resource_discovery_research_copilot(request): return _run(request, feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, scale="prospective high-throughput", require_approval=True, require_federation=False)
__all__=["ResourceCopilotRequest","ResourceCopilotReceipt","worldgen_throughput_resource_discovery_research_copilot_manifest","run_worldgen_throughput_resource_discovery_research_copilot"]
